src/kontrol/kdist/cheatcodes.md

```diff
@@ -426,7 +426,30 @@ This rule returns a symbolic boolean value being either 0 (false) or 1 (true).
          <k> #call_foundry SELECTOR _ => . ... </k>
          <output> _ => #buf(32, ?WORD) </output>
       requires SELECTOR ==Int selector ( "freshBool()" )
-       ensures #rangeBool(?WORD)
+       ensures #rangeBool(?WORD2)
+       [preserves-definedness]
+```
+
+#### `freshBytes` - Returns a single symbolic boolean.
+
+```
+function freshBytes(uint256) external returns (bytes memory);
+```
+
+`foundry.call.freshBytes` will match when the `freshBytes` cheat code function is called.
+This rule returns a symbolic boolean value being either 0 (false) or 1 (true).
```

Comment:
I don't think that's what's being returned?

Reply:
Fixed.